tests/rgbpp/spore/launch/2-create-cluster.ts

```diff
@@ -0,0 +1,101 @@
+import { BtcAssetsApiError, genCreateClusterCkbVirtualTx, sendRgbppUtxos } from 'rgbpp';
+import { isMainnet, collector, btcAddress, btcDataSource, btcKeyPair, btcService } from '../../env';
+import { CLUSTER_DATA } from './0-cluster-info';
+import { transactionToHex } from 'rgbpp/btc';
+import {
+  appendCkbTxWitnesses,
+  buildRgbppLockArgs,
+  generateClusterCreateCoBuild,
+  sendCkbTx,
+  updateCkbTxWithRealBtcTxId,
+} from 'rgbpp/ckb';
+import { getFastestFeeRate, readStepLog, writeStepLog } from '../../shared/utils';
+
+// Warning: Before runing this file, please run 1-prepare-cluster.ts
+const createCluster = async ({ ownerRgbppLockArgs }: { ownerRgbppLockArgs: string }) => {
+  console.log(btcAddress);
+  const { retry } = await import('zx');
+  await retry(20, '10s', async () => {
+    const feeRate = await getFastestFeeRate();
+    console.log('feeRate = ', feeRate);
+
+    const ckbVirtualTxResult = await genCreateClusterCkbVirtualTx({
+      collector,
+      rgbppLockArgs: ownerRgbppLockArgs,
+      clusterData: CLUSTER_DATA,
+      isMainnet,
+      ckbFeeRate: BigInt(2000),
+    });
```

Comment:
### suggestion
Also follow the best practice -> `saveCkbVirtualTxResult` in the tests.

- ref: https://github.com/ckb-cell/rgbpp-sdk/issues/192